plugins/image-prioritizer/readme.txt

```diff
@@ -0,0 +1,58 @@
+=== Image Prioritizer ===
+
+Contributors:      wordpressdotorg
+Requires at least: 6.4
+Tested up to:      6.5
+Requires PHP:      7.2
+Stable tag:        0.1.0
```

Comment:
I'd go with 1.0.0 - what have we done n the past for new plugins?

Reply:
Optimization Detective started with 0.1.0, as did Embed Optimizer. I'd keep it below 1.x since it's still experimental. Once something graduates from being experimental, then I think it makes sense to go 1.x.